patches/react-native+0.61.5.patch

```diff
@@ -53,3 +67,10 @@ index 40c0cda..646f137 100644
    }
    return ancestor == shadowView;
  }
+diff --git a/node_modules/react-native/scripts/.packager.env b/node_modules/react-native/scripts/.packager.env
```

Comment:
This change seems like it's not related, will we need this as well?

Reply:
Thanks for catching that!